src/librustc/hir/mod.rs

```diff
@@ -385,7 +385,7 @@ impl PathSegment {
     }
 
     pub fn generic_args(&self) -> &GenericArgs {
-        if let Some(ref args) = self.args {
+        if let Some(ref args) = self.args{
```

Comment:
Accidental misformat?

Reply:
Yes, it was `param` before but I was told to change it back.
I'll add the whitespace back

Reply:
Changed